android/cli/commands/_build.js

```diff
@@ -3683,8 +3683,7 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 		}
 	}, this);
 
-	
-	if (this.realTargetSDK >= 24 && !finalAndroidManifest.application.resizeableActivity) {
+	if (this.realTargetSDK >= 24 && !("resizeableActivity" in finalAndroidManifest.application)) {
```

Comment:
Generally we prefer `hasOwnProperty()` over `in` like this:

    finalAndroidManifest.application.hasOwnProperty('resizeableActivity')

Also, single quotes, not double quotes.